hive/envs/__init__.py

```diff
@@ -1,11 +1,13 @@
 from hive.envs.base import BaseEnv, ParallelEnv
 from hive.envs.env_spec import EnvSpec
 from hive.envs.gym_env import GymEnv
-
-try:
-    from hive.envs.minigrid import MiniGridEnv
-except ImportError:
-    MiniGridEnv = None
+from minigrid import (
+    FlatObsWrapper,
+    FullyObsWrapper,
+    ImgObsWrapper,
+    RGBImgObsWrapper,
+    RGBImgPartialObsWrapper,
+)
```

Comment:
Can you update with changes from dev branch and move these changes to gym/gym_wrappers.py, inside of a try block?
Like:
```
try:
   importing and registering minigrid wrappers
except:
   ...
```